internal/runbits/proxyreader/proxyreader.go

```diff
@@ -0,0 +1,49 @@
+package proxyreader
+
+import (
+	"io"
+
+	"github.com/ActiveState/cli/internal/errs"
+)
+
+type ByIncrementer interface {
+	IncrBy(int)
+}
+
+var _ io.Reader = &ProxyReader{}
+
+// ProxyReader wraps around a reader and calls the incrementer function on every read reporting the number of bytes read
+type ProxyReader struct {
```

Comment:
This does not seem like it belongs in `runbits`.

Reply:
I moved it to `internal/proxyreader`.  I couldn't really think of any other package that it could fit with...